appium-examples/README.md

```diff
@@ -2,9 +2,9 @@
 This folder contains Appium examples
 
 ## Examples
-- [Using biometric login on Sauce Labs](./src/test/java/biometric_login)
-- [Using image injection on Sauce Labs](./src/test/java/image_injection)
-- [Android native app test](./src/test/java/com/realdevice/unifiedplatform/AndroidNativeAppTest.java)
```

Comment:
@eyaly please follow the standard folder structure outlined [here](https://github.com/saucelabs-training/demo-java/blob/master/CONTRIBUTING.md#repository-structure)

       |-- appium-examples (Java module)
        |-- src
            |-- main
            |-- test
                |-- java
                    |-- com.emusim (Java package)
                        |-- SomeEmusimExample.java
                    |-- com.realdevice.unifiedplatform (Java package)
                        |-- SomeRealDeviceExample.java
                    |-- com.realdevice.legacy (Java package)
                        |-- ...

Reply:
I still think that the main folders should be nativeApp and webApp because or you test your web app or your native app. After that, you can choose between emuson and RDC . But to be able to push the changes faster - I changed back to the original structure.